Parse/Public/ParseObject.cs

```diff
@@ -1263,7 +1264,7 @@ where item.CanBeSerialized
     /// retrieved and <paramref name="key"/> is not found.</exception>
     /// </summary>
     public T Get<T>(string key) {
-      return (T)ParseClient.ConvertTo<T>(this[key]);
+      return (T)Conversion.ConvertTo<T>(this[key]);
```

Comment:
In this case it doesn't make sense, as we'll need to do an additional check for `!=` null and then throw an exception in that case (I guess we can craft a ClassCastException, but... why?)